protocol/avdl/keybase1/simple_fs.avdl

```diff
@@ -571,4 +572,35 @@ protocol SimpleFS {
   void simpleFSSubscribeNonPath(union{ null, TLFIdentifyBehavior } identifyBehavior, string subscriptionID, SubscriptionTopic topic, int deduplicateIntervalSecond);
 
   void simpleFSUnsubscribe(union{ null, TLFIdentifyBehavior } identifyBehavior, string subscriptionID);
+
+  record DownloadInfo {
+    string downloadID;
+    KBFSPath path;
+    string filename;
+    Time startTime;
+    boolean isRegularDownload;
+  }
+  record DownloadState {
+    string downloadID;
+    double progress;
+    Time endEstimate;
+    string localPath;
+    string error;
+    boolean done;
+    boolean canceled;
+  }
+  record DownloadStatus {
+    array<string> regularDownloadIDs; // the ones showing up in the GUI footer
+    array<DownloadState> states; // all downloads including non-regular ones
+  }
+  // returns download ID. isRegularDownload causes moving to the Downloads
+  // folder. Save on mobile and share to other apps use isRegularDownload=false.
```

Comment:
So "regular" means something that shows up in the footer?  Why don't "save on mobile" count as "regular"?  Seems like the name could be a bit more descriptive.

Reply:
> So "regular" means something that shows up in the footer?

Yes that's correct, plus that it's moved to the Downloads folder after it's done. It feels to be purely a GUI term,. I'll clarify in the comment.